configs/config.hs

```diff
@@ -1,9 +1,30 @@
+-- This file is part of purebred
+-- Copyright (C) 2017 Róman Joost
+--
+-- purebred is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU Affero General Public License as published by
+-- the Free Software Foundation, either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU Affero General Public License for more details.
+--
+-- You should have received a copy of the GNU Affero General Public License
+-- along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-
-Example configuration, which uses more mutt-alike keybindings
+Example configuration, currently used for testing which demonstrates various
+ways to overwrite the configuration.
 -}
 import Purebred
+import qualified Data.ByteString.Lazy as B
```

Comment:
I'd prefer `L` for qualifying `Data.ByteString.Lazy`.

Reply:
Done :)